style.css

```diff
@@ -32,72 +29,61 @@ i{
 }
 
 #logo{
-    font-size: 30px;
     font-weight: 600;
-    margin-left: 20px;
     cursor: context-menu;
-    position: relative;
+    display: flex;
+    align-items: center;
 }
 
 .logo-icon{
-    position: absolute;
-    width: 65px;
-    height: 60px;
-    left: 0px;
-    top: 2px;
+    width: 5rem;
+    height: 4rem;
+    align-self: center;
 }
 
 .name{
     color: var(--primary);
-    margin-left: 60px;
     font-weight: bolder;
-    font-size: 60px;
+    font-size: 4rem;
     font-family:Georgia, 'Times New Roman', Times, serif;
 }
 .Home{
     display: flex;
-    font-size: 20px;
+    font-size:4rem;
     justify-content:flex-end;
     color:var(--primary) ;
 }
 .Home>a{
     margin: 10px;
-    padding: 15px;
+    padding:1rem;
     font-size: large;
 }
-.conten{
-    margin-top: 5%;
-    cursor:context-menu;
-    width: 70%;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-}
-.conten>p{
-    width: 100%;
-    font-size: 70px;
-    line-height: normal;
+.content-container{
+    margin: 3rem;
+    width:80%;
+    align-self: center;
     text-align: center;
+    font-size: 5rem;
 }
 .btn1{
     font-size: large;
     background-color: var(--primary);
     padding: 10px;
     border: none;
     color: var(--ternary);
-    width: 150px;
-    border-radius:20px;
-    margin: 15px;
+    width: 10rem;
+    border-radius:2rem;
+    margin: 8px;
 }
 .btn2{
```

Comment:
Don't make class name like btn1 or btn2 give some meaningful nameor you can use naming convention like making it btn-1 or btn-2 or btn- primary or btn-secondary etc.